### PR TITLE
Sensor show page

### LINKED ISSUE #88: As a visitor, when I visit a sensor show page I see that sensor's last 5 readings, a button to edit the sensor, and a search function to get the last x days of readings

### LINKED ISSUE #131: All links to the api are called through the env variable ENV['BE_URL']

### PR DESCRIPTION
**Pull request by lkriffell**
closes #88 
closes #131

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -19,10 +19,67 @@
...
+
 ul {
   list-style-type: none;
 }
 
 .icon {
   margin: 5px;
-}
\ No newline at end of file
+}
```
Comment (LHJE) [inline app/assets/stylesheets/application.scss:85]: Great job with this!

File: app/poros/garden_health.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    @created_at = data[:attributes][:created_at]
+    determine_type
+  end
+
+  def determine_type
+    if @reading_type == 'light'
+      @reading = "#{((@reading / 770) * 100).round(2)}%"
+    end
+  end
+end
```
Comment (LHJE) [inline app/poros/garden_health.rb:22]: very cool

File: app/views/sensors/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                        <% time = reading.created_at.to_time.to_s.split(" ")[1] %>
+                        <%= reading.created_at.to_date %> <%= time %><span class="pull-right"><%= reading.reading %></span><br>
+                    <% end %>
+                  </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    <% end %>
+</center>
```
Comment (LHJE) [inline app/views/sensors/show.html.erb:51]: I'm floored, this is great!

File: spec/features/gardens/show_spec.rb [hunk 4]
```diff
@@ -184,6 +184,9 @@
       sensors = File.read('spec/fixtures/sensors.json')
       stub_request(:get, "#{ENV['BE_URL']}/api/v1/gardens/3/sensors").to_return(status: 200, body: sensors)
 
+      sensor = File.read('spec/fixtures/sensor.json')
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/sensors/#{@sensor1[:id]}").to_return(status: 200, body: sensor)
```
Comment (LHJE) [inline spec/features/gardens/show_spec.rb:188]: 'https://solar-garden-be.herokuapp.com' should be replaced with "#{ENV['BE_URL']}".

**Review by LHJE — CHANGES_REQUESTED** [review]
Looks great! But what is this "solar-garden-be" folder?  It keeps popping up in PR's and I'm not sure where it came from...

File: app/poros/garden_health.rb (hunk 2 above)
Comment (AngelaGuardia) [reply app/poros/garden_health.rb:22]: Nice!

File: app/views/sensors/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,51 @@
+<center>
+  <h3>You're <%= @sensor.sensor_type %> sensor</h3>
```
Comment (AngelaGuardia) [inline app/views/sensors/show.html.erb:2]:
```suggestion
  <h3>Your <%= @sensor.sensor_type %> sensor</h3>
```

**Review by AngelaGuardia — APPROVED** [review]
Massive PR Logan! Well done. I think I may have created some of the same files as you so we may have some merger conflicts depending on which PR gets merged first. Added one suggestion for spelling on one of your views.

File: spec/facades/sensor_facade_spec.rb [hunk 6]
```diff
@@ -15,7 +15,7 @@
     params = {id: 1}
     sensor = File.read('spec/fixtures/sensor.json')
 
-    stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/sensors/#{params[:id]}").to_return(status: 200, body: sensor, headers: {})
+    stub_request(:get, "#{ENV['BE_URL']}/api/v1/sensors/#{params[:id]}").to_return(status: 200, body: sensor, headers: {})
```
Comment (LHJE) [inline spec/facades/sensor_facade_spec.rb:18]: Love it!

**Review by LHJE — APPROVED** [review]
Great job!

**Pull request merged**